dockerfiles/Dockerfile.osm-bootstrap

```diff
@@ -1,2 +1,2 @@
 FROM gcr.io/distroless/static
-COPY osm-bootstrap /
+COPY . /
```

Comment:
Can the CRDs instead be staged to the `osm-bootstrap` folder as a part of the build? This would make it consistent with the other Dockerfiles (osm-controller, osm-injector).

Reply:
as discussed offline, the crds are a part of the osm-bootstrap folder